draft-pardue-moq-qlog-moq-events.md

```diff
@@ -946,31 +956,85 @@ MOQTSubscribeDone = {
 ~~~
 {: #subscribedone-def title="MOQTSubscribeDone definition"}
 
-### MOQTMaxSubscribeId
+### MOQTPublish
 
 ~~~ cddl
-MOQTMaxSubscribeId = {
-  type: "max_subscribe_id"
-  subscribe_id: uint64
+MOQTPublish = {
+  type: "publish"
+  request_id: uint64
+  track_alias: uint64
```

Comment:
I think we should keep the same order as the draft so `track_alias` would come after `track_name`.